lang/cs/Org.Apache.REEF.Tang/Util/ReflectionUtilities.cs

```diff
@@ -591,5 +591,27 @@ private static bool HasInjectableConstructor(Type type)
             }
             return isInjectable;
         }
+
+        public static int GetDefaultValueOfIntNamedParameter(Type namedParamType)
```

Comment:
No need this method and Tang will get default value automatically for you if it is not bound.